Assets/Scripts/Quiz/RandMark.cs

```diff
@@ -1,39 +1,23 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
 using static SoundManager;
 
 public class RandMark : MonoBehaviour
 {
     [SerializeField, Header("�����̖���")]
-    private GameObject _RandMarkTest;
-    [SerializeField,Tooltip("�J�E���g")]
-    private int _Count;
-    [SerializeField, Tooltip("�J�E���g")]
-    private float _Time;
-    // Start is called before the first frame update
-    private void Update()
+    private GameObject _randMarkTest;
```

Comment:
```suggestion
    private GameObject _randMark;
```
テストじゃないと思うので正式版にしましょう

Reply:
_randMarkTestは最初はTestでちゃんと動くか確かめてましたが、
Testの部分を消すの忘れてました。

Reply:
上記の修正内容は
すみません。これに関しては目を通してなかったです。
なので、今はTestを削除して_randMarkに変更済みです。